plugin-maven/src/test/java/com/diffplug/spotless/maven/shell/ShellTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.maven.shell;
+
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.diffplug.spotless.maven.MavenIntegrationHarness;
+
+@Disabled // Disabled due to GHA not having shfmt
```

Comment:
Please let me know if I missed some detail that enables these types of formatters (ones that need to shell out) to run in GitHub action workflows. I'd be happy to add whatever is needed to get this to run in GHA.

Reply:
I replaced `@Disabled` with `@ShfmtTest`. If you run `gradlew test` these tests will not run. They only run when you run `./gradlew testShfmt` to run them.

- https://github.com/diffplug/spotless/blob/main/gradle/special-tests.gradle

If you want them to run on CI, you should add them with the npm test

https://github.com/diffplug/spotless/blob/797a665465429729b5308ee77f392c170c57242e/.github/workflows/ci.yml#L79-L81

- change it to `./gradlew testNpm testShfmt`

The only tricky part is getting shfmt onto the CI runner. Not that hard though. Feel free to open a PR for it and mess around.

Reply:
Perfect! Thank you for the clarification.